### DISCUSSION TITLE
Logging Progress Externally

### DISCUSSION THREAD

**Discussion by onekmilesbehind** [staxrip/staxrip#1195] [Q&A] [answered]
I could be missing something very obvious here, but does anyone know of a way to log the progress/status of the active encode to an external file periodically, say once every minute? From what I can see this type of data isn't included in the default log.

My end-goal is to be able to take that data and have another app scrape that into my Home Assistant instance as a sensor value. I have encodes run for 3+ days and so I'm trying to find a better way to monitor the encoder output than remoting in to my encoder machine.

Here are the stats I'm referring to, and thanks in advance for any help!

![Progress Bar](https://github.com/staxrip/staxrip/assets/17152356/d6272ee3-b0b8-460f-81da-571a57d4eefe)

**Comment by Dendraspis** [comment, marked as answer]
Nope, that's not supported at the moment and I don't have a workaround for that right now because there is nothing usable for your scenario.

Seems like you would like to open a *feature request*.  😅

**Reply by onekmilesbehind** [reply]
Awesome, thanks for confirming--wasn't sure if I was missing something, but I figure it's an odd use-case anyway.

I'll open a feature request!
